src/include/sof/ipc/driver.h

```diff
@@ -45,6 +45,11 @@ enum task_state ipc_platform_do_cmd(struct ipc *ipc);
  */
 void ipc_platform_complete_cmd(struct ipc *ipc);
 
+/**
+ * \brief Tell host we have received the last IPC command.
```

Comment:
Let's add a clear comment that the platform specific implementation is allowed to invalidate the incoming IPC mailbox (the data is either already consumed or copied).